patchkit-launcher-qt-src/src/launcherworker.h

```diff
@@ -35,7 +35,7 @@ class LauncherWorker : public QThread
     void cancel();
 
     bool canStartPatcher() const;
-    void startPatcher();
+    void startPatcher(bool isOffline = false);
```

Comment:
Avoid default parameters, it is very important one and it should be always explicitly specified.